adal/tests/Test.ADAL.NET.UIAutomation/XamarinDroidMSALTests.cs

```diff
@@ -81,26 +81,9 @@ public void AcquireTokenTest()
         [Test]
         public void PromptBehavior_Consent_SelectAccount()
         {
-            var labData = LabUserHelper.GetLabResponseWithDefaultUser();
-
-            // 1. Acquire token with uiBehavior set to consent 
-            _msalMobileTestHelper.AcquireTokenInteractiveTestHelper(
-                xamarinController,
-                labData,
-                CoreUiTestConstants.UIBehaviorConsent);                        
-
-            // 2. Switch ui behavior to "select account"
-            _msalMobileTestHelper.SetUiBehavior(xamarinController, CoreUiTestConstants.UIBehaviorSelectAccount);
-
-            // 3. Hit Acquire Token directly since we are not changing any other setting
-            xamarinController.Tap(CoreUiTestConstants.AcquireTokenID);
-
-            // 4. The web UI should display all users, so click on the current user
-            xamarinController.Tap(labData.User.Upn, XamarinSelector.ByHtmlValue);
-
-            // 5. Validate token again
-            _msalMobileTestHelper.CoreMobileTestHelper.VerifyResult(xamarinController);
+            var labResponse = LabUserHelper.GetLabResponseWithDefaultUser();
 
+            _msalMobileTestHelper.PromptBehaviorTestHelper(xamarinController, labResponse);
```

Comment:
>_msalMobileTestHelper.PromptBehaviorTestHelper(xamarinController, labResponse); [](start = 12, length = 79)

Does this set consent by default? That is not obvious to me.

Reply:
yes. however, it should allow you to set whatever behavior you want. ill update.